internal/exp/metrics/identity/stream.go

```diff
@@ -33,3 +33,7 @@ func OfStream[DataPoint attrPoint](m Metric, dp DataPoint) Stream {
 type attrPoint interface {
 	Attributes() pcommon.Map
 }
+
+func Compare[T interface{ Hash() hash.Hash64 }](a, b T) int {
+	return int(a.Hash().Sum64() - b.Hash().Sum64())
+}
```

Comment:
I can't see this being used in this git diff, so I'm wondering why it's needed 🤔

Reply:
ahh, it can be used with https://pkg.go.dev/slices#SortFunc, which I don't appear to do anymore. still useful, vote to keep

Reply:
I'd vote to remove unused code, but no strong opinions :)